app/presenters/vacancy_presenter.rb

```diff
@@ -121,11 +121,11 @@ def flexible_working
   # rubocop:enable Rails/OutputSafety
 
   def working_pattern
-    model.working_pattern.sub('_', ' ').humanize
+    model.working_patterns.join(', ').sub('_', ' ').humanize
```

Comment:
Doesn't `humanize` already include the underscore substitution?

Reply:
Yea, good catch 👍 In fact when looking at it now, I think I need to apply `humanize` to each of the working patterns not both together. I'll add a commit